pallets/payment-streams/src/lib.rs

```diff
@@ -573,11 +583,14 @@ pub mod pallet {
             // Check that the extrinsic was executed by the root origin
             ensure_root(origin)?;
 
+            let current_tick = Self::get_current_tick();
+
             // Execute checks and logic, update storage
             Self::do_update_dynamic_rate_payment_stream(
                 &provider_id,
                 &user_account,
                 new_amount_provided,
+                current_tick,
```

Comment:
`do_update_dynamic_rate_payment_stream` can lookup the `current_tick` on its own, it doesn't need to receive it as a parameter.